src/helm.exec.ts

```diff
@@ -4,22 +4,24 @@ import { helm as logger } from './logger';
 import * as YAML from 'yamljs';
 import * as _ from 'lodash';
 import * as tmp from 'tmp';
-import * as extension from './extension';
 import * as helmrepoexplorer from './helm.repoExplorer';
 import * as helm from './helm';
 import { showWorkspaceFolderPick } from './hostutils';
-import { shell as sh, ShellResult, ExecCallback } from './shell';
+import { shell as sh, ShellResult, ExecCallback, shell } from './shell';
```

Comment:
Double import here -> `shell as sh` and `shell`

Reply:
Oof, good catch.  Thanks!